rdr_client/hpo_assigner.py

```diff
@@ -13,31 +15,35 @@
 
 import csv
 import logging
-
+import sys
 from main_util import get_parser, configure_logging
 
 from client import Client, HttpException, client_log
 
-_ORG_PREFIX = 'Organization/'
-
 
 def main(client):
   num_no_change = 0
   num_updates = 0
   num_errors = 0
+  pairing_list = ['site', 'organization', 'awardee']
+
+  if client.args.pairing not in pairing_list:
+    sys.exit('Pairing must be one of site|organization|awardee')
+
   with open(client.args.file) as csvfile:
     reader = csv.reader(csvfile)
     for line in reader:
       try:
-        participant_id, hpo = [v.strip() for v in line]
+        participant_id, new_pairing = [v.strip() for v in line]
+        pairing = client.args.pairing
```

Comment:
You can assign this just once, outside the loop.

Also, rename to "pairing_key"?